### PR TITLE
Link error gio on windows fix

### PR DESCRIPTION
**Pull request by daa84**
Fix gtk-rs/gtk#517
I'am not good with gir and copy this from glib crate, so hope this is ok.

### CODE REVIEW

File: Gir.toml [hunk 1]
```diff
@@ -43,6 +43,9 @@ manual = [
 [[object]]
 name = "Gio.*"
 status = "generate"
+    [[object.function]]
+    pattern = "unix_.+"
+	cfg_condition = "unix"
```
Comment (EPashkin) [inline Gir.toml]: Same indent as `pattern`

**Comment by EPashkin** [comment]
Thanks, @daa84 This is what we need
Please, change last commit message to "Regen" as It don't generated new functions ;)

**Comment by daa84** [comment]
Done :smile:

**Comment by EPashkin** [comment]
@GuillaumeGomez CI passed.

**Comment by GuillaumeGomez** [comment]
Thanks!

**Pull request merged**